src/features/author.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable-next-line no-param-reassign */
```

Comment:
not next line - for the entire file (as in comments.ts)
And fix in other files

![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/638f7662-c439-46ab-8159-e14b9936fcfd)